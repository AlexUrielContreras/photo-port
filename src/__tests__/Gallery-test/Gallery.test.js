import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Gallery from '../../components/Gallery';

const portrait = { name: 'portrait', description: 'Portrait of people in my life'};

afterEach(cleanup);

describe('Gallery is rendering', () => {
    it('renders', () => {
        render(<Gallery currentCategory={portrait} />)
    });

    it('atches snapshot', () => {
        const { asFragment } = render(<Gallery currentCategory={portrait} />)

        expect(asFragment()).toMatchSnapshot()
    });

    it('renders', () => {
        const { getByTestId } = render(<Gallery currentCategory={portrait} />);

        expect(getByTestId('h1tag')).toHaveTextContent('Portrait')
    })
})