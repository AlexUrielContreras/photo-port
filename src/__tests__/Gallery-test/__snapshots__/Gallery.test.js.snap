// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gallery is rendering atches snapshot 1`] = `
<DocumentFragment>
  <section>
    <h1
      data-testid="h1tag"
    >
      Portrait
    </h1>
    <p>
      Portrait of people in my life
    </p>
    <div>
      <div
        class="flex-row"
      />
    </div>
  </section>
</DocumentFragment>
`;
